frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -350,6 +350,7 @@ export type ListPageHeaderProps = {
   title: string;
   helpText?: React.ReactNode;
   badge?: React.ReactNode;
+  hideFavoriteButton?: boolean;
```

Comment:
Please also update example in `frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts`. Once you update it, run `yarn generate-plugin-sdk-docs` to update the docs.